app/controllers/metrics_controller.rb

```diff
@@ -250,7 +261,8 @@ def update_watchlist_configuration
     begin
       # handle nil blocklist in models
       config = WatchlistConfiguration.update_watchlist_configuration_for_course(course_name,
-                                                                                params[:blocklist])
+                                                                                params[:blocklist],
+                                                                                params[:allow_ca])
```

Comment:
The comment at the start of the method should be updated to reflect that `allow_ca` is a required param

Reply:
make sense